testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LevelOfAssuranceFlowTest.java

```diff
@@ -0,0 +1,272 @@
+package org.keycloak.testsuite.forms;
```

Comment:
It might be good if you could add header comments the same as other newly added source files like AuthenticatorUtil.java .